src/components/js/Homepage.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+import Form from './Form';
+import '../css/Homepage.css';
+import Todos from './Todos';
+import NavLinks from './NavLinks';
+
+export default function Home() {
+  const [todo, setTodo] = useState('');
+  const [storage, setStorage] = useState(JSON.parse(localStorage.getItem('todos')) ? JSON.parse(localStorage.getItem('todos')) : []);
+  const handleChange = (event) => {
+    setTodo(event.target.value);
+  };
+
+  const handleRemove = (event) => {
+    const { id } = event.target.parentNode.parentNode.parentNode.parentNode;
+    const updatedStore = storage.filter((element) => element.id.toString() !== id);
+    localStorage.setItem('todos', JSON.stringify(updatedStore));
+    setStorage(updatedStore);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    if (todo) {
+      const form = event.target;
+      const obj = { id: new Date().getTime().toString(), title: todo, completed: false };
+      setStorage([...storage, obj]);
+      console.log([...storage]);
```

Comment:
- [ ] Make sure you get rid of all console logs before publishing.

Reply:
Thanks for reviewing. I have gotten rid of all console logs.